### ISSUE TITLE
404 page 

### ISSUE THREAD

**Issue by daedraMex** [swetankraj/dark-netflix-api#8]
If you introduce a diferent endpoint, the program will redirect you to 404 page

**Comment by swetankraj** [comment]
@Any28Flo Yeah I was going to do it, but glad you pointed it out. If you'd like to work on this I'm happy to help.

**Comment by devanshuyadav** [comment]
Hey @swetankraj, I'd like to work on it. Let me know if the issue is still open to working upon.
